sdk/template/azure-template2/README.md

```diff
@@ -0,0 +1,70 @@
+# Azure Template Package client library for C++
```

Comment:
I don't understand why a second template package is required. What benefit are we getting from this that a single test package can't give us?

Reply:
The steps are different first pushing a port and then pushing another port to the same branch. To simulate the differences in doing so we need 2 ports to test publishing. This is different from other package repositories where you can push packages as needed and don't need to make atomic updates. 

In the interest of getting these changes in I'll back out template2 for now and we can revisit it in https://github.com/Azure/azure-sdk-for-cpp/issues/1651